contracts/RestrictedNORI.sol

```diff
@@ -882,7 +895,7 @@ contract RestrictedNORI is
       scheduleId,
       quantityToRevoke
     );
-    // slither-disable-next-line calls-loop no option for batch send on ERC777
+    // slither-disable-next-line calls-loop send individually for each schedule after calculating quantity to revoke
```

Comment:
it's because `batchSend` would have to be called outside of this function in its calling function `batchReleaseUnrevokedTokenAmounts` , but we need `_revokeUnreleasedTokens` to do the work of figuring out what the batch send amounts would be.  When we overhaul that batch revoke function, as there are already todos written for, we can use a batch send at the end of it all.  I can remove the slither disable